shell/common/engine.cc

```diff
@@ -244,7 +244,6 @@ Engine::RunStatus Engine::Run(RunConfiguration configuration) {
 }
 
 void Engine::BeginFrame(fml::TimePoint frame_time, uint64_t frame_number) {
-  TRACE_EVENT0("flutter", "Engine::BeginFrame");
```

Comment:
Can you add comments to these deletions to explain why they are okay?  Just looking at the PR it's hard for me to know if this is acceptable or not.

Reply:
Sure.

This one is redundant with `Animator::BeginFrame`. It adds no additional information and never really shows anything valuable.